src/sentry/models/groupsnooze.py

```diff
@@ -52,28 +52,24 @@ def is_valid(self, group=None, test_rates=False):
             raise ValueError
 
         if self.until:
-            if self.until > timezone.now():
```

Comment:
all of this just changes the behavior so if you specified multiple attributes here it would reflect them

its not really needed, but in the future maybe we'll want it